service/src/main/resources/HTMLTemplate.html

```diff
@@ -525,24 +525,28 @@ <h3 class="heading heading--two">Child
         case_data.children.firstChild is not empty and
         case_data.children.firstChild.livingSituation is not empty %}
           <span class="u-block">{{ case_data.children.firstChild.livingSituation }}</span>
-          {% if case_data.children.firstChild.address is not empty %}
-            {% if case_data.children.firstChild.address.AddressLine1 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine1 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.AddressLine2 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine2 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.AddressLine3 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine3 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.PostTown is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.PostTown }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.PostCode is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.PostCode }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.Country is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.Country }}</span>
+          {% if case_data.children.firstChild.detailsHidden is not empty and case_data.children.firstChild.detailsHidden == "Yes" %}
```

Comment:
Story says:

> And the current living situation is answered with 'Living with Respondents'

I cannot see condition around `case_data.children.firstChild.livingSituation`

Reply:
good q, let me double check this is correct

Reply:
this should be hide for any address entered. AC to be updated.